tests/test_add_id.py

```diff
@@ -56,6 +67,8 @@ def test_basic_id(self, single_partition_dataset):
                 "doc_id-0000000004",
             ]
         )
+        if is_cudf_available and isinstance(actual_ids, cudf.Series):
+            actual_ids = actual_ids.to_pandas()
```

Comment:
Can we make sure we are actually entering this `if` block in the GPU case? Maybe:
```
if is_cudf_available:
    assert isinstance(actual_ids, cudf.Series)
    actual_ids = actual_ids.to_pandas()
```
?

Reply:
So we actually run this cell for both cuDF and Pandas.  So we cant really ` assert isinstance(actual_ids, cudf.Series)`. 

We check `is_cudf_available` above now for GPU cases so i think that case is checked so i think we should be fine.